proposals/security_and_privacy.md

```diff
@@ -0,0 +1,277 @@
+
+This risk is mitigated by the fact that endpoint IoT devices
+are generally not exposed directly on the internet, but
+are mediated by gateways and cloud services.
+
+### Does this specification enable new script execution/loading mechanisms? 
+
+Yes.  It enables WoT Things to run applications based
+on ECMAScript, using a constrained API, and with suitable
+restrictions, such as no direct access to private
+security data.
+
+However, a mechanism for loading new scripts is not defined
+and is currently considered to be out of scope.
+Our architecture assumes the scripts running inside a WoT Thing
+are trusted implementations provided by the manufacturer
+of the device and installed using a suitable OTA update
+mechanism; but this update mechanism is not defined in our
+architecture and is also currently considered to be out of scope.
+
+### Does this specification allow an origin access to a user’s location?
+
+As noted above in the PII response, in theory a device
+location could be encoded in a WoT Thing Description using
+an extension vocabulary, and this could in some cases be
+associated with a user.
+Location information, however, is neither a standard nor
+a required part of the architecture.
+
+A manufacturer of a device could also define an
+interaction that returns the
+location of the device.
+We do not provide constraints on this, since it is usually
+the _purpose_ of IoT devices to provide access to sensor data.
+
+However, we do provide mechanisms to control access to
+interactions and a manufacturer can use these to
+prevent accidental disclosure to unauthorized users.
+A user can also inspect the WoT Thing Description to
+determine whether the device returns location information,
+assuming the manufacturer has not obfuscated the purpose
+of interactions.  
+
+### Does this specification allow an origin access to sensors on a user’s device?
+
+Yes.  Many other sensors can be attached to an IoT 
+device and we do not constrain these in any way.
+
+It is the responsibility of the user to only connect and
+provision devices they trust, to select devices with
+good security, and manage the data that is
+collected appropriately.
+
+The metadata provided in the WoT Thing Description
+can be helpful in this regard since the security mechanisms
+supported by a device can be easily inspected and,
+via semantic annotations, the type of data provided
+by the device can be determined.
+
+### Does this specification allow an origin access to aspects of a user’s local computing environment?
+
+No.  Actually, there is no executable content
+included in data or metadata provided by WoT Things.
+The only operations that a WoT Server will perform
+on behalf of a WoT Client are those that it chooses
+to support via its exposed interactions (network API).
+
+### Does this specification allow an origin access to other devices? 
+
+Yes, as discussed above.  A WoT Thing can act as a gateway to 
+other WoT Things or to other non-IP protocols.
+
```

Comment:
Access to other devices is one of the primary use cases of WoT.

Reply:
Yep, I probably should say something strong here.   I will steal your sentence.

Reply:
Fixed up, I think.  Expanded explanation of "Zero-Trust"; this should go into the main document, too.   And into our Best Practices doc...